package org.mangala.portfolio.portfolio.adapter.web.dto;

import jakarta.validation.constraints.Size;
import lombok.Data;

@Data
public class UpdatePortfolioRequestDTO {
    @Size(max = 100, message = "Portfolio name must be at most 100 characters")
    private String name;

    @Size(max = 500, message = "Description must be at most 500 characters")
    private String description;
}
